server/store/store.go

```diff
@@ -153,6 +162,10 @@ func (s *SQLStore) addColumn(tableName, columnName, columnDefinition string) err
 }
 
 func (s *SQLStore) Init() error {
+	if err := s.createTable("msteamssync_subscriptions", "subscriptionID VARCHAR(255) PRIMARY KEY, type VARCHAR(255), msTeamsTeamID VARCHAR(255), msTeamsChannelID VARCHAR(255), msTeamsUserID VARCHAR(255), secret VARCHAR(255), expiresOn BIGINT"); err != nil {
```

Comment:
Has this been tested with both Postgres and MySQL?

Reply:
I'm running it through the tests, so the table creation should work in both or fail in the CI.